examples/python/sandbox_remote.py

```
@@ -0,0 +1,301 @@
+    ssh_host = os.getenv("SSH_HOST", "localhost")
+    ssh_user = os.getenv("SSH_USER", "ubuntu")
+    ssh_key = os.getenv("SSH_KEY_FILE", "~/.ssh/id_rsa")
+    
+    print(f"Connecting to {ssh_user}@{ssh_host}")
+    
+    try:
+        sandbox = SSHSandbox(
+            host=ssh_host,
+            user=ssh_user,
+            key_file=ssh_key,
+            working_dir="/tmp/praisonai_demo"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/praisonai_demo"

(S108)